pages/common/hledger-add.md

```diff
@@ -0,0 +1,28 @@
+# hledger add
+
+> Record new transactions with interactive prompting in the console.
+> More information: <https://hledger.org/hledger.html#add>.
+
+- Record new transactions, saving to the default journal file:
+
+`hledger add`
+
+- Add transactions to `2024.journal`, but also load `2023.journal` for completions:
+
+`hledger add --file 2024.journal --file 2023.journal`
+
+- Provide answers for the first four prompts:
+
+`hledger add today 'best buy' expenses:supplies '$20'`
+
+- Show add's options and documentation with $PAGER:
+
+`hledger add --help`
+
+- Show add's documentation with info or man if available:
+
+`hledger help add`
+
+- Show add's documentation on the web:
+
+`open 'https://hledger.org/hledger#add'`
```

Comment:
Not sure whether or not we should add this example... First, `open` is non-standard and second, the link is already present in the page as the more information link. I personally would remove the example.